libcontainerd/queue_unix_test.go

```diff
@@ -0,0 +1,36 @@
+package libcontainerd
+
+import (
+	"testing"
+	"time"
+)
+
+func TestSerialization(t *testing.T) {
+	var (
+		q             queue
+		serialization = 1
+	)
+
+	q.append("aaa", func() {
+		//simulate a long time task
+		time.Sleep(10 * time.Millisecond)
+
+		if serialization != 1 {
```

Comment:
Perhaps good to use testify for these (like in https://github.com/moby/moby/blob/16380b3ee68c0da2fda036d7980129241957220c/pkg/promise/promise_test.go#L7-L13)